samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -316,6 +320,8 @@ DEFINE_uint64(nireq, 0, infer_requests_count_message);
 /// @brief Number of streams to use for inference on the CPU (also affects Hetero cases)
 DEFINE_string(nstreams, "", infer_num_streams_message);
 
+DEFINE_string(cpu_core_ids, "", infer_cpu_core_ids_message);
```

Comment:
Please extend https://github.com/openvinotoolkit/openvino/blob/master/tests/samples_tests/smoke_tests/test_benchmark_app.py to run the new flag and the list of cmd args in https://github.com/openvinotoolkit/openvino/blob/master/docs/articles_en/learn-openvino/openvino-samples/benchmark-tool.rst.